doc/command_ref.rst

```diff
@@ -1797,9 +1799,12 @@ to packages added from commandline, installed packages, or packages from reposit
 ``module_hotfixes=true`` in their .repo file.
 
 Disabling of modular filtering is not recommend, because then the system could move into broken
-state. To disable modular filtering for particular repository specify ``module_hotfixes=true``
+state! To disable modular filtering for particular repository specify ``module_hotfixes=true``
 in the .repo file or add ``--setopt=<repo_id>.module_hotfixes=true`` into into command.
```

Comment:
The paragraph has quite a number of grammar/spelling issues, let's fix it? I'd actually prefer not using the exclamation mark, but keep it in if you want to :slightly_smiling_face: 

> Disabling of modular filtering is not recommend**ed**, because **it could cause** the system **to get** into **a** broken state. To disable modular filtering for **a** particular repository**,** specify ``module_hotfixes=true``
in the .repo file or **use** ``--setopt=<repo_id>.module_hotfixes=true``.